src/reader.rs

```diff
@@ -202,7 +202,8 @@ impl Reader {
             sr_iters.push_back(iter);
         }
 
-        DbIterator::new(range.clone(), mem_iter, l0_iters, sr_iters).await
+        // TODO(flaneur): load max committed seq here
```

Comment:
Remind me: is seq purely in memory, or does L0 contains sequences as well? I recall some discussion about this, but it was a while ago.

Reply:
yes, the L0 SST is a 1:1 dumping of memtable, it will also include multiple versioned data as well.

Reply:
Gotcha. In that case, shouldn't `DbIterator::new` take a sequence number corresponding to the snapshot's last sequence number? Or since the two are guaranteed to be the same, it's not needed?

Reply:
yes indeed, the seq num should be passed from `DbIterator::new`, this can be placed in a standalone pr for issue #489